### PR TITLE
Add cnv_abnormal metric

### PR DESCRIPTION
**Pull request by avlitman**
cnv_abnormal added to telemeter: https://github.com/openshift/cluster-monitoring-operator/pull/2291.
This pr adding the metric to telemeter server metrics allow list.

### CODE REVIEW

File: configuration/telemeter/metrics.json [hunk 1]
```diff
@@ -86,8 +87,8 @@
   "{__name__=\"eo_es_misconfigured_memory_resources_info\"}",
   "{__name__=\"eo_es_redundancy_policy_info\"}",
   "{__name__=\"eo_es_storage_info\"}",
-  "{__name__=\"hypershift_cluster_cores\"}",
   "{__name__=\"hostedcluster:hypershift_cluster_vcpus:max\"}",
+  "{__name__=\"hypershift_cluster_cores\"}",
```
Comment (simonpasquier) [inline configuration/telemeter/metrics.json:91]: not sure why the generation moved this line but it fails the CI.

**Comment by juzhao** [comment]
do we need to add the metric to resources/services/telemeter-template.yaml? see PR: https://github.com/rhobs/configuration/pull/686

File: configuration/telemeter/metrics.json (hunk 1 above)
Comment (avlitman) [reply configuration/telemeter/metrics.json]: I revert it let's see if it pass

Comment (avlitman) [reply configuration/telemeter/metrics.json]: I reverted everything and then run `make all` I didn't even added cnv_abnormal manually

**Comment by avlitman** [comment]
closing since https://github.com/rhobs/configuration/pull/690

**Pull request closed without merging**